notebooks/257-blip-diffusion-subject-generation/README.md

```diff
@@ -0,0 +1,33 @@
+# Subject-driven image generation and editing using BLIP Diffusion and OpenVINO
+![](https://github.com/salesforce/LAVIS/raw/main/projects/blip-diffusion/teaser-website.png)
```

Comment:
please test in colab and add badges if it works

Reply:
It does not work in Colab because of RAM limit.